zkevm-circuits/src/evm_circuit/execution/returndatasize.rs

```diff
@@ -70,14 +72,12 @@ impl<F: Field> ExecutionGadget<F> for ReturnDataSizeGadget<F> {
     ) -> Result<(), Error> {
         self.same_context.assign_exec_step(region, offset, step)?;
         let return_data_size = block.get_rws(step, 1).stack_value();
-        self.return_data_size.assign(
+        self.return_data_size.assign_lo(
             region,
             offset,
-            Some(
-                return_data_size.to_le_bytes()[..N_BYTES_U64]
-                    .try_into()
-                    .expect("could not encode return_data_size as byte array in little endian"),
-            ),
+            Value::known(from_bytes::value(
+                &return_data_size.to_le_bytes()[..N_BYTES_U64],
+            )),
```

Comment:
Suggestion:
```rust
self.return_data_size.assign_u64(region, offset, return_data_size.to_u64());
```

The general rule I follow for this suggestion is to:
- Avoid calling `to_le_bytes()` manually because it adds verbosity
- If we want to assign a Word to a smaller signal, instead of discarding bytes, doing a conversion like `to_u64()` which panics when the value is too big.

Reply:
Yes, `assign_lo` is an old api, and after few api iteration, now we have assign_u64. 

I removed assign_lo api in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1411/commits, along with some places to switch to proper `assign_u64`